cpython-unix/build-main.py

```diff
@@ -53,11 +53,12 @@ def main():
         help="Target host triple to build for",
     )
 
+    optimizations = {"debug", "noopt", "pgo", "lto", "pgo+lto"}
     parser.add_argument(
-        "--optimizations",
-        choices={"debug", "noopt", "pgo", "lto", "pgo+lto"},
+        "--options",
+        choices=optimizations.union({f"freethreaded+{o}" for o in optimizations}),
```

Comment:
Nit: I might just construct this inline on line 56, rather than splitting the construction between there and here.

Reply:
It seems a bit annoying to construct in one line. Are you suggesting I just enumerate them all manually?

Reply:
I was suggesting:
```python
optimizations = {"debug", "noopt", "pgo", "lto", "pgo+lto"}
optimizations = optimizations.union({f"freethreaded+{o}" for o in optimizations})
parser.add_argument(..., choices=optimizations)
```

Reply:
Ah thanks for clarifying. It'd be `options = optimizations.union(...)` and I'm hesitant to introduce another local variable (basically just personal preference).